source/includes/_supplier_link.md

```diff
@@ -0,0 +1,67 @@
+# Supplier Links
+
+**Transaction ID (required)**
```

Comment:
Its not really required if we tac it to the end of the url, right? 
Is it worth having this param in there, since we'll add it to the url every time? 

It will not always be on the end of the url.  The link hash will be appended (when available).

Reply:
![image](https://user-images.githubusercontent.com/20267274/136288309-de111644-2cae-4b84-bb09-bfe2c45edbc2.png)

This was just a copy-paste job from campaigns out into its own section but I agree, I enclosed a screenshot of the new copy but what I ended up doing is removing the reference to it being required, and the verbiage of it being on the end of the URL and added an example of how the URL should look upon entry.